config.go

```diff
@@ -48,16 +48,22 @@ const (
 	ccProxy = "cc"
 )
 
+const (
+	ccShim = "cc"
```

Comment:
Oh - reading on, I see the test that shows `shim.cc`. Now I'm really confused - rather than using top-level entities `shim.cc` and `proxy.cc`, wouldn't it be better to create some sort of `cc` namespace and then define `shim` and `proxy` inside that? Can that be done with toml?